crates/typescript-tests/run.sh

```diff
@@ -48,9 +50,9 @@ for filename in *.ts; do
     # Copy over every line EXCEPT for the import statements (since "no-modules" has no modules to import).
     grep -v -w "import" "$filename" > "$path"
     # Then replace all of the instances of "wbg" (the namespace all the tests use to import the *.d.ts files from "--target web") with "wasm_bindgen" (the namespace the `no-modules` *.d.ts files use).
-    sed -i "s/wbg\./wasm_bindgen./g" "$path"
-    sed -i "s/wbg /wasm_bindgen /g" "$path"
-    sed -i "s/wbg\[/wasm_bindgen[/g" "$path"
+    sed -i'.orig' "s/wbg\./wasm_bindgen./g" "$path"
+    sed -i'.orig' "s/wbg /wasm_bindgen /g" "$path"
+    sed -i'.orig' "s/wbg\[/wasm_bindgen[/g" "$path"
```

Comment:
The existing command doesn't work on macOS because sed on macOS treats the first arg after `-i` as suffix.

`sed -i ''` may not work on Linux, and `sed -i''` doesn't seem to work on macOS, so I decided to just give it a suffix, which seems to work on both.